l10n/en/products/vpn/shared.ftl

```diff
@@ -75,3 +75,6 @@ vpn-shared-pricing-save-percent = Save { $percent }%
 vpn-shared-pricing-total = { $amount } total
 
 ##
+
+# message shown to visitors who try to sign-in without an active subscription.
+vpn-shared-sub-not-found = Oops, we can’t find your subscription.
```

Comment:
This is just placeholder copy for now, will update once we have the final string.